### PR TITLE
Consistency updates

### PR DESCRIPTION
**Pull request by rikkepaul**

### CODE REVIEW

File: applications/frontend/src/App.tsx [hunk 1]
```diff
@@ -1,73 +1,89 @@
...
+      ingredients: ingredients,
+    });
+    try {
+      await fetch("http://localhost:8000/checkLiveness", {
+        method: "GET",
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }).then(() => setLoading(false));
+    } catch (error) {
```
Comment (martehvi) [inline applications/frontend/src/App.tsx:39]: did we figure out if this resulting in no loading symbol?

File: applications/frontend/src/App.tsx [hunk 2]
```diff
@@ -1,73 +1,89 @@
...
+    const requestBody = JSON.stringify({
+      ingredients: ingredients,
+    });
+    try {
+      await fetch("http://localhost:8000/checkLiveness", {
+        method: "GET",
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }).then(() => setLoading(false));
```
Comment (martehvi) [inline applications/frontend/src/App.tsx]: the "setLoading(false)" can be removed, should only be set as done after the recipe call

File: applications/frontend/src/App.tsx [hunk 3]
```diff
@@ -1,73 +1,89 @@
...
+        .then((data) => setRecipe(data))
+        .finally(() => setLoading(false));
+    } catch (error) {
+      console.error("Failed to execute the second fetch call:", error);
+    }
+  }
+
+  return (
+    <>
+      <Box>Andrea's Magic Cookbook</Box>
```
Comment (martehvi) [inline applications/frontend/src/App.tsx]: change name 🥴

File: 01-docker-compose/README.md [hunk 4]
```diff
@@ -10,23 +10,23 @@ Before we delve into the `docker-compose.yml` file lets first run our Backend an
...
    - The `-p` flag exposes a port on your local machine and maps it to a port on the docker container. The mapping uses the format, `<host-port>:<container-port>`.
 
 ### Backend
 
-Now that you got the frontend up and running, lets do the same for the backend. Change directories into the backend folder and spin up the docker file there as well.
+Now that you got the frontend up and running, lets do the same for the backend. Change directories into the backend folder (`cd ../backend/`) and spin up the docker file there as well.
 
-1. Start off with building a docker image by running: `docker build -t <image-tag-name> -f <docker-filename> .`
-2. Next lets run the image we just built by using: `docker run -p 8000:8000 -t <image-tag-name>`
+1. Start off with building a docker image by running: `docker build -t my-awesome-backend -f backend.dockerfile .`
```
Comment (martehvi) [inline 01-docker-compose/README.md]: talked about reverting these changes, waiting to improve PR until that is done

File: applications/frontend/src/App.tsx (hunk 1 above)
Comment (rikkepaul) [reply applications/frontend/src/App.tsx:39]: After removing line 38 this was no longer an issue! 🥳

File: 01-docker-compose/README.md [hunk 5]
```diff
@@ -12,23 +12,38 @@ Before we delve into the `docker-compose.yml` file lets first run our Backend an
...
    - The `-p` flag exposes a port on your local machine and maps it to a port on the docker container. The mapping uses the format, `<host-port>:<container-port>`.
 
+<dispaly>
+<summary>
+
+Hint 🔍 - What is _&lt;image-tag-name&gt;_ and _&lt;docker-filename&gt;_ ?
+
+</summary>
+
+- _&lt;image-tag-name&gt;_: This is the desired name or tag you want to assign to your Docker image. Choose a meaningful and descriptive name to easily identify the image. If you do not provide this than docker will autogenerate a random name for your image. If you do not specify a tag or want to find out what your image is named try running `docker images`. If you did not specify a value the _TAG_ column will have the _none_ value. So, when you then in the second command, `docker run` need to refer to the image you can either use the tag you provided when creating the image or the _IMAGE ID_ (tip: you only need a uniqe part of the image id prefix with minimum two character length).
```
Comment (martehvi) [inline 01-docker-compose/README.md]: "than docker will.." => "th**e**n docker will.."

**Review by martehvi — APPROVED** [review]

**Pull request merged**